src/main/java/euteam/cleanmachine/model/facility/Program.java

```diff
@@ -6,18 +6,16 @@
 import javax.persistence.Id;
 
 @Entity
-public class Program {
-
+public abstract class Program {
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
+
     private String name;
+
     private double duration;
-    private double cost;
 
-    public Program() {
-        this.cost = 3;
-    }
```

Comment:
Here is the missing 3.
3+4+4 = 11

Reply:
Oo I see, sorry removed it because I didn't see why it should be 3

Reply:
Oo I see, sorry removed it because I didn't see why it should be 3